services/discovery/.gitignore

```diff
@@ -0,0 +1,244 @@
+
```

Comment:
I created this subdirectory using `sam init`. I'm not convinced this is the final directory structure; we can refactor after.